practice-two/src/js/script.js

```diff
@@ -166,6 +166,35 @@ const saveData = () => {
     });
   }
   localStorage.setItem(UsersKey, JSON.stringify(existingEntries));
+  renderUser();
+}
+
+/**
+ * Show user data in table
+ */
+const renderUser = () => {
+  let user = `  
```

Comment:
user? Variable name is not suitable name. It should be template, pls also update function name too